src/docs/carousel.md

```diff
@@ -1,5 +1,8 @@
 # Carousel
 
+## Content Disclaimer
+This UX pattern has been renamed from **Paging Panel**.
```

Comment:
Can you change it to the following: This UX design pattern was previously called Paging Panel.